### DISCUSSION TITLE
[BUG] npm ERR! code ENOENT

### DISCUSSION THREAD

**Discussion by iiAlanii** [discordjs/discord.js#6088] [Q&A] [answered]
PS C:\Users\NuL\Documents\BOT> npm install discord.js
npm ERR! code ENOENT
npm ERR! syscall spawn git
npm ERR! path git
![Annotation 2021-07-10 040139](https://user-images.githubusercontent.com/85262775/125160771-9be52700-e133-11eb-813a-6cd032c5baa6.png)

npm ERR! errno -4058
npm ERR! enoent Error while executing:
npm ERR! enoent undefined ls-remote -h -t ssh://git@github.com/discordjs/discord.js.git
npm ERR! enoent
npm ERR! enoent 
npm ERR! enoent spawn git ENOENT
npm ERR! enoent This is related to npm not being able to find a file.
npm ERR! enoent

npm ERR! A complete log of this run can be found in:

**Comment by vladfrangu** [comment, marked as answer]
You are missing git from your machine! You'll need to install it according to your OS instructions

**Reply by kyranet** [reply]
In case of Windows, which seems to be your OS: https://gitforwindows.org
